test/test-exec-deserialization.py

```diff
@@ -178,6 +178,37 @@ def test_removal(self):
 
         self.assertTrue(not os.path.exists(self.output_file))
 
+    def test_issue_6533(self):
+        unit = "test-issue-6533.service"
+        unitfile_path = "/run/systemd/system/{}".format(unit)
```

Comment:
I think `tearDown` should remove `/run/systemd/system/test-issue-6533.service` together with `self.unitfile_path` and `self.output_file`. Anyway, the patch looks good to me and I'm ready to merge it as is.

Reply:
You are right. But for now  I'd leave it as is. I think it would be better to rewrite test so that we'd have some ```cleanup``` list that we append to and tearDown will remove all files in the list, but I'd like to do that in separate PR.